src/app/core/export/query.service.spec.ts

```diff
@@ -618,6 +618,12 @@ describe("QueryService", () => {
     expect(result).toEqual([maleChild.gender]);
   });
 
+  it("does not throw an error if no query is provided", () => {
+    return expectAsync(
+      service.queryData(undefined, new Date(), new Date())
+    ).toBeResolved();
```

Comment:
what is the expected result here actually? Should this be null/undefined/empty or something or passing through the values of the previous level? In the UI, will this just show a label and nothing in the count/value spot?

Reply:
The value is a empty object (I added this to the test) what is displayed is just the label, without a value in this line.